### PR TITLE
Update release notes - Host Reports

### PR DESCRIPTION
**Pull request by lzap**
SSIA

FYI the plugin is not in 3.2 yet we are finishing migration rake task today and then releasing a new version that is the final one.

### CODE REVIEW

File: _includes/manuals/3.2/1.2_release_notes.md [hunk 1]
```diff
@@ -2,12 +2,24 @@
 
 ### Headline features
 
+#### Host reports
```
Comment (ekohl) [inline _includes/manuals/3.2/1.2_release_notes.md]: I don't think this should be a headline feature if there is no installer support. I also think we shouldn't introduce headline features in a bugfix release.

IMHO we should either postpone the release (and make sure everything gets in) if we want to make it a headline feature or we release it in 3.2.1 as an optional new feature. I'm leaning to the latter.

Comment (lzap) [reply _includes/manuals/3.2/1.2_release_notes.md]: Ok how about Preview feature? I will link the known bug as well.

Comment (lzap) [reply _includes/manuals/3.2/1.2_release_notes.md]: I moved it down and marked them as a preview feature.

Comment (ares) [reply _includes/manuals/3.2/1.2_release_notes.md]: I think the "preview" makes sense if we provide instructions how to install. We want to let users know they can start playing with that. Also that prepares the ground for the release with 3.3.

**Comment by ekohl** [comment]
By now the reports plugin has been retired so I'm closing this.

**Pull request closed without merging**